scripts/windows/deploy.one.cmd

```diff
@@ -34,8 +34,6 @@ REM
     set rdst=\\%machine%\%rdst_dir%
     @mkdir %rdst%    
     xcopy /F /Y /S %src_dir% %rdst%
-    COPY /Y %bin_dir%\7z.exe %rdst%
```

Comment:
The monitor will install 7z itself. And I thought this line is used when we unziped the package on the remote machine while we now unzip the package locally.